app/assets/stylesheets/main.css.scss

```diff
@@ -52,3 +52,6 @@ $compatibility-high-color: #84C93F;
 @import 'styles/pages/home';
 @import 'styles/pages/styles';
 
+// Import modular CSS
+@import "modules/**/**/*";
+@import "modules/**/**/**/*";
```

Comment:
Yeah that's what I meant. Like I said, I'm not sure how the import works, so this might be a nuance that I missed.